multimap/src/main/java/org/infinispan/multimap/api/package-info.java

```diff
@@ -0,0 +1,6 @@
+/**
+ * Embedded Multimap Cache.
+ *
+ * @public
+ */
+package org.infinispan.multimap.api;
```

Comment:
This package should only be present in the commons Jar. Additional API "extensions" for embedded and remote should be further qualified. We must not have classes present in the same package in different jars.

Reply:
is it ok if I moved them to api.embedded ?

Reply:
so I have package org.infinispan.multimap.api.embedded;